src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/generate_mails_form.html.twig

```diff
@@ -65,6 +65,9 @@
             <div class="col-sm">
               {{ form_errors(generateMailsForm.theme) }}
               {{ form_widget(generateMailsForm.theme) }}
+              {% if generateMailsForm.theme.vars.disabled %}
+              <small class="form-text">{{ 'No mails were detected in any theme folder so this field is disabled.'|trans({}, 'Admin.Design.Help') }}</small>
```

Comment:
Hello! Please write _No emails were detected in any theme folder so this field is disabled._ instead, thank you.

Reply:
Done